Mods/src/PuppiMod.cc

```diff
@@ -92,33 +101,43 @@ Int_t PuppiMod::GetEtaBin(const PFCandidate *cand)
 }
 
 //--------------------------------------------------------------------------------------------------
-Double_t PuppiMod::Chi2fromDZ(Double_t dz)
+Double_t
+PuppiMod::Chi2fromDZ(Double_t dz) const
 {
   Double_t probPV = ROOT::Math::normal_cdf_c(fabs(dz),fTrackUncertainty) * 2.0;
   Double_t probPU = 1 - probPV;
   Double_t chi = 0;
-  if (probPU == 1) 
+  if (probPU == 1)
     chi = 100;                                // This doesn't exactly match CMSSW, but I like this better
-  else 
+  else
     chi = TMath::ChisquareQuantile(probPU,1);
-  return pow(chi,2);
+  return chi * chi;
 }
 
 //--------------------------------------------------------------------------------------------------
-void PuppiMod::SlaveBegin()
+void
+PuppiMod::SlaveBegin()
 {
+  // Read the configuration file
+  if (fEtaConfigName == "") {
+    SendError(kAbortAnalysis, "SlaveBegin", "Config name for Puppi not given.");
+    return;
+  }
+
   // Prepare the storage array for the PuppiParticles
   fPuppiParticles = new PFCandidateArr(16);
   fPuppiParticles->SetName(fPuppiParticlesName);
   PublishObj(fPuppiParticles);
 
-  // Read the configuration file
-  if (fEtaConfigName == "")
-    SendError(kAbortAnalysis, "SlaveBegin", "Config name for Puppi not given.");
+  Info("SlaveBegin", "Reading PUPPI config file " + fEtaConfigName);
```

Comment:
Use Info() where possible instead of cout, so that the text output is formatted and is easy to read.